NativeUI/Items/UIMenuSliderItem.cs

```diff
@@ -0,0 +1,169 @@
+﻿using System.Collections.Generic;
+using System.Drawing;
+
+namespace NativeUI
+{
+    public class UIMenuSliderItem : UIMenuItem
+    {
+        protected Sprite _arrowLeft;
+        protected Sprite _arrowRight;
+
+        protected UIResRectangle _rectangleBackground;
+        protected UIResRectangle _rectangleSlider;
+        protected UIResRectangle _rectangleDivider;
+        
+        protected int _value = 0;
+        protected int _max = 100;
+        protected int _multiplier = 5;
+
+
+        /// <summary>
+        /// Triggered when the slider is changed.
+        /// </summary>
+        public event ItemSliderEvent OnSliderChanged;
+
+
+        /// <summary>
+        /// The maximum value of the slider.
+        /// </summary>
+        public int Maximum
+        {
+            get
+            {
+                return _max;
+            }
+            set
+            {
+                if (_value > value)
+                {
+                    _value = value;
+                }
+            }
+        }
+        /// <summary>
+        /// Curent value of the slider.
+        /// </summary>
+        public int Value
+        {
+            get
+            {
+                return _value;
+            }
+            set
+            {
+                if (value > _max)
+                    _value = 100;
```

Comment:
Should be _value = _max;

Reply:
This is a leftover when the max value was fixed to 100. Done in 41a50b620d5f11938f5c94edf7899f8824e40910.